pages/consulting/[filename].tsx

```diff
@@ -50,6 +50,11 @@ export default function ConsultingPage(
     recaptchaKey: props.env["GOOGLE_RECAPTCHA_KEY"],
   };
 
+  data.consulting.seo = {
+    ...data.consulting.seo,
+    title: data.consulting.seo?.title || data.consulting.title
```

Comment:
Can you check where we are using `consulting.title`?

I think it could be removed now we have the title in the SEO.

If we dont need it, remove it.

Reply:
Done - Removed consulting.title and used consulting.seo.title instead